plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -57,7 +63,33 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 		} );
 		await page.locator( '#title' ).fill( virtualProductName );
 		await page.locator( '#_regular_price' ).fill( productPrice );
+		await page.locator( '#_sale_price' ).fill( salePrice );
 		await page.locator( '#_virtual' ).click();
+
+		// Fill in a product description
+		await page
+			.frameLocator( '#content_ifr' )
+			.locator( '.wp-editor' )
+			.fill( productDescription );
+
+		// Fill in SKU
+		await page.getByText( 'Inventory' ).click();
+		await page.locator( '#_sku' ).fill( productSKU );
```

Comment:
I've followed @lanej0's [suggestion](https://github.com/woocommerce/woocommerce/pull/43710#discussion_r1454084102) and used `getBy*` instead of `locator` wherever possible. For the SKU form, you can try the same I did, per @veljkho's advice, [here](https://github.com/woocommerce/woocommerce/pull/43710/files#diff-4d8b4fa2d3fc09e284b17868562b01cf7bc2d5294477590b31e45f0f01277ebbR156), it should work 🤞

Reply:
This looks good @Stojdza.

The failing CI check you have is just because you have to add a changelog entry.  If you go to the `plugins/woocommerce` folder and run `pnpm --filter=plugin-woocommerce run changelog add` you can do that, commit it and it should resolved the failing check.

Reply:
Thanks @lanej0, I've just added this via CLI.